### ISSUE TITLE
Xaml files outside the current csproj's directory do not appear in the VS solution explorer

### ISSUE THREAD

**Issue by mjrousos** [dotnet/core#2010]
In a .NET Core 3 (3.0.100-alpha1-009689) WPF project, including a XAML file from outside the csproj's directory will include the file in the project but not display it in Visual Studio's solution explorer, making it difficult to navigate to during development time.

Sample csproj demonstrating the issue:

```xml
<Project Sdk="Microsoft.NET.Sdk.Wpf">
  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>netcoreapp3.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <!-- These files will appear in VS's solution explorer -->
    <ApplicationDefinition Include="App.xaml" />
    <Page Include="MainWindow.xaml" />
    
    <!-- This file will *not* appear since it is in a parent directory -->
    <Page Include="../XamlResources.xaml">

    </Page>
  </ItemGroup>

  <ItemGroup>
    <FrameworkReference Include="Microsoft.DesktopUI" />
  </ItemGroup>
</Project>
```

The recommended approach to porting WPF apps to .NET Core is to create another (netcoreapp3.0) head project in a directory next to the one containing the original NetFx csproj, so I expect this issue to come up a lot while porting WPF apps.

**Comment by diverdan92** [comment]
@Pilchie - can you take a look?

**Comment by Pilchie** [comment]
This happens because the solution explorer doesn't know what name/path to display the file under the project folder with the `Link` metadata.  Adding `Link="XamlResources.xaml"` to the `<Page>` element makes it show up.

@lifengl I wonder if we could consider a default for Link metadata if it doesn't appear in the file?  Also @dsplaisted and @nguerrera and @davkean.

**Comment by dsplaisted** [comment]
Yes, we should consider adding `Page` to the item types that get automatic Link metadata, either in the WPF SDK or just by default.  This issue can be moved to dotnet/cli or dotnet/sdk.

(I would move it myself but the issue mover I normally use seems to be broken).

**Comment by Pilchie** [comment]
Same here (with https://github-issue-mover.appspot.com/). 😦 

```
Error creating new issue: NullError: Cannot call "ez" on null
```

Anyone know who owns that issue mover?
